src/main/java/org/jabref/logic/importer/fileformat/PdfMergeMetadataImporter.java

```diff
@@ -124,8 +128,12 @@ public ParserResult importDatabase(Path filePath) throws IOException {
                 }
             }
         }
-
-        entry.addFile(new LinkedFile("", filePath, StandardFileType.PDF.getName()));
+        if (workingDir != null) {
+            Path relativizedFilePath = workingDir.relativize(filePath);
+            entry.addFile(new LinkedFile("", relativizedFilePath, StandardFileType.PDF.getName()));
+        } else {
+            entry.addFile(new LinkedFile("", filePath, StandardFileType.PDF.getName()));
+        }
```

Comment:
Please add JavaDoc explaning the param `workingDir` and state when this can be null.

I was about to write that you should switch the if branches to avoid negation (and use workingDir == null), but this one is IMHO more readable

```suggestion
        if (workingDir != null) {
            filePath = workingDir.relativize(filePath);
        }
        entry.addFile(new LinkedFile("", filePath, StandardFileType.PDF.getName()));
```